vendor/k8s.io/kubernetes/pkg/kubectl/cmd/cmd.go

```diff
@@ -161,6 +161,7 @@ __custom_func() {
    * events (aka 'ev')
    * endpoints (aka 'ep')
    * horizontalpodautoscalers (aka 'hpa')
+   * images
```

Comment:
Oh, and I'd rather see a method that pulls and caches discovery information.  If the server responds in time or the cache is close enough to up to date, use the actual discovered resources.